<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaCliente</class>
 <widget class="QWidget" name="VistaCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="l_nome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_cognome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_indirizzo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_telefono">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_email">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="l_codice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modifica">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifica</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_elimina">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
